Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_028.PlaintextPassword.ps1

```diff
@@ -10,6 +10,7 @@ ConvertTo-SecureString -String 'D$oodle3333' -AsPlainText -Force)  New-Mailbox -
 ConvertTo-SecureString -String 'Doodle$555555' -AsPlainText -Force PS C:\&gt; Add-AzureRmServiceFabricNodeType -ResourceGroupName 'Group1' -Name 'someContosoCluster' -NodeType 'n2' -Capacity 5 -VmUserName 'someName' -VmPassword $pwd</dev:code>         <dev:remarks>           <maml:para>This command will add a new NodeType 'n2' w
 ConvertTo-SecureString -String "666666" -AsPlainText -Force PS c:\&gt; add-AzServiceFabricClusterCertificate -ResourceGroupName 'Group2' -Name 'someContosoCluster'  -CertificateSubjectName 'Contoso.com'  -CertificateOutputFolder 'c:\test' -CertificatePassword $pwd</dev:code>         <dev:remarks>           <maml:para>This command will
 ConvertTo-SecureString -String '6ood137777' -AsPlainText -Force))  #Set-TargetResource "Present" "test" "C:\Builds\vs_ultimate.exe" ""  "/quiet /noweb" "HKLM:\SOFTWARE\Wow6432Node\Microsoft\DevDiv\vs\Servicing\12.0\ultimate1" (New-Object System.Management.Automation.PSCredential -ArgumentList 'domain\account', (ConvertTo-SecureStr"
+"commandToExecute": "[concat('powershell.exe -ExecutionPolicy bypass \"& ./artifact.ps1 -StringParam ''', parameters('stringParam'), ''' -SecureStringParam (ConvertTo-SecureString ''', parameters('securestringParam'), ''' -AsPlainText -Force) -IntParam ', parameters('intParam'), ' -BoolParam:$', parameters('boolParam'), ' -FileContentsParam ''', parameters('fileContentsParam'), ''' -ExtraLogLines ', parameters('extraLogLines'), ' -ForceFail:$', parameters('forceFail'), '\"')]"
```

Comment:
Is this a positive or negative test case? I was expecting to see a negative test case added to the "# These should not be matched" section below.

Reply:
@marmegh Technically, this regex does not generate a new "negative" space. It also increases the granularity of the positive space. For this command, the earlier regex would have returned `''` as the match, whereas this regex would return `, parameters('securestringParam`),`. Both are false positives, but the second is more accurate.